function/spring-computer-vision-function/build.gradle

```diff
@@ -0,0 +1,7 @@
+dependencies {
+    api "ai.djl.spring:djl-spring-boot-starter-autoconfigure:$djlSpringVersion"
+    api "ai.djl.spring:djl-spring-boot-starter-tensorflow-auto:$djlSpringVersion"
+    api "ai.djl.spring:djl-spring-boot-starter-pytorch-auto:$djlSpringVersion"
+    api "ai.djl.spring:djl-spring-boot-starter-mxnet-auto:$djlSpringVersion"
```

Comment:
Don't they have a BOM for all of these dependencies?

The Dependa Bot is not going to understand this module config: we have to go the way as it is done in the `dependencies.gradle`, for example, for the `springIntegrationAws`  property.

Reply:
Apparently only mavenBom "ai.djl:bom:$djlVersion" exist not for the spring boot auto-configuraitons.

Reply:
Any chances that we can raise a GH issue against that project, so they'll address it this or other way?